### PR TITLE
add CURLOPT_PROXY_ISSUERCERT for consistency

### PR DESCRIPTION
**Pull request by gvollant**
CURLOPT_ISSUERCERT exists but not CURLOPT_PROXY_ISSUERCERT 

the openssl vtls is already compatible with issuercert against proxy

### CODE REVIEW

File: docs/libcurl/opts/Makefile.inc [hunk 1]
```diff
@@ -258,6 +258,8 @@ man_MANS =                                      \
   CURLOPT_PROXY_CAPATH.3                        \
   CURLOPT_PROXY_CRLFILE.3                       \
   CURLOPT_PROXY_KEYPASSWD.3                     \
+  CURLOPT_PROXY_ISSUERCERT.3                    \
+  CURLOPT_PROXY_ISSUERCERT_BLOB.3               \
```
Comment (jay) [inline docs/libcurl/opts/Makefile.inc:262]: This line ends in "\ " but must end in "\" no quotes. Other than that looks fine will check CI after fix is committed.

Comment (gvollant) [reply docs/libcurl/opts/Makefile.inc]: ok, it is fixed

**Comment by bagder** [comment]
This commit doesn't add any code that *uses* these options, is that already in place?

**Comment by gvollant** [comment]
in openssl.c we have already

    if(SSL_SET_OPTION(issuercert) || SSL_SET_OPTION(issuercert_blob)) {

and SSL_SET_OPTION will uses proxy parameters if needed

**Comment by bagder** [comment]
Thanks!

**Pull request closed without merging**